src/Database/Esqueleto/PostgreSQL.hs

```diff
@@ -1,22 +1,30 @@
+{-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE OverloadedStrings
  #-}
 -- | This module contain PostgreSQL-specific functions.
 --
 -- /Since: 2.2.8/
 module Database.Esqueleto.PostgreSQL
   ( arrayAggDistinct
+  , arrayAggDistinct'
   , arrayAgg
+  , arrayAgg'
   , arrayRemove
+  , arrayRemoveNull
+  , array
   , stringAgg
   , chr
   , now_
   , random_
+  , toChar
```

Comment:
The additions in this module look great! Would you be willing to write some tests for the `unsafeSqlFunction` bits?